src/Tasks/Microsoft.NET.Build.Tasks/msbuildExtensions/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.targets

```diff
@@ -0,0 +1,17 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <_TargetFrameworkVersionWithoutV>$(TargetFrameworkVersion)</_TargetFrameworkVersionWithoutV>
+    <_TargetFrameworkVersionWithoutV Condition="$(TargetFrameworkVersion.StartsWith('v'))">$(TargetFrameworkVersion.Substring(1))</_TargetFrameworkVersionWithoutV>
+
+    <MicrosoftNETBuildExtensionsTasksAssembly Condition="'$(MicrosoftNETBuildExtensionsTasksAssembly)' == '' AND '$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)\tools\netcoreapp1.0\Microsoft.NET.Build.Extensions.Tasks.dll</MicrosoftNETBuildExtensionsTasksAssembly>
+    <MicrosoftNETBuildExtensionsTasksAssembly Condition="'$(MicrosoftNETBuildExtensionsTasksAssembly)' == ''">$(MSBuildThisFileDirectory)\tools\net46\Microsoft.NET.Build.Extensions.Tasks.dll</MicrosoftNETBuildExtensionsTasksAssembly>
+
+    <!-- Include conflict resolution targets for NETFramework and allow other frameworks to opt-in -->
+    <ResolveAssemblyConflicts Condition="'$(ResolveAssemblyConflicts)' == '' AND '$(TargetFrameworkIdentifier)' == '.NETFramework' AND '$(UsingMicrosoftNETSdk)' != 'true'">true</ResolveAssemblyConflicts>
```

Comment:
Those were measurements I made early on of just the task that was being added.  I went back and measured the targets in their current form, results are here: https://gist.github.com/ericstj/b56c6a430e79e6768f93551b66219d22

This is for a packages.config-based desktop project building in visual studio.  It's showing zero-impact to the hello-world case, and 16 ms cold / 0 ms warm to the top-10 packages (20 assemblies) case.  I've tried a few different variations of this but can't get numbers worse than this for these projects, so I'm fairly confident that we'll be able to keep the impact low.